src/main/scala/io/iohk/ethereum/blockchain/sync/FastSyncNodesRequestHandler.scala

```diff
@@ -31,20 +31,20 @@ class FastSyncNodesRequestHandler(
 
     val hashesToRequest = (nodeData.values.indices zip receivedHashes) flatMap { case (idx, valueHash) =>
       requestedHashes.find(_.v == valueHash) map {
-        case StateMptNodeHash(hash) =>
-          handleMptNode(hash, nodeData.getMptNode(idx))
+        case StateMptNodeHash(_) =>
+          handleMptNode(nodeData.getMptNode(idx))
 
-        case ContractStorageMptNodeHash(hash) =>
-          handleContractMptNode(hash, nodeData.getMptNode(idx))
+        case ContractStorageMptNodeHash(_) =>
```

Comment:
generally I prefer to use `_: ContractStorageMptNodeHash` that way if at some point someone adds new field to `ContractStorageMptNodeHash` this match will still compile